builds/misc/templates/mqtt-perf-setup.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
         sshEndpoint: iotedge-mqtt-perf-amd64-conn
         runOptions: commands        
         commands: |
-          docker stop mqtt-broker ; docker system prune -a -f
+          echo $(brokerImageName) > brokerImageName.txt ; docker stop mqtt-broker ; docker system prune -a -f
```

Comment:
why do we need to do this?

Reply:
We have a "Cleanup" step down the pipeline, and it needs an image name to re-create the container. The problem is, to pass env var from one job to another, you need to write a an ugly echo command on every step. Saving the image name into the local file seems a good workaround.

```
echo "##vso[task.setvariable variable=brokerImageName;]$brokerImageName"